airflow/kubernetes/pod_launcher.py

```diff
@@ -176,16 +176,20 @@ def monitor_pod(self, pod, get_logs):
             self.log.info('Container %s has state %s', pod.metadata.name, State.RUNNING)
             time.sleep(SleepConfig.BASE_CONTAINER_RUNNING_POLL)
         if self.extract_xcom:
-            result = self._extract_xcom(pod)
-            self.log.info(result)
-            result = json.loads(result)
-        istio_shut_down_initiated = False
+            try:
+                result = self._extract_xcom(pod)
+                self.log.info(result)
+                _result = json.loads(result)
+            except Exception as e:
+                self.log.error(f"Failed to extract XCOM: {e}")
+                result = None
```

Comment:
I don't love big except blocks like this, but we do want to run istio no matter what..